src/rust/ensogl/example/src/rendering_error.rs

```diff
@@ -0,0 +1,121 @@
+//! Example scene showing simple usage of a shape system.
+
```

Comment:
I usually mark comments as resolved when I am sure that I have fixed them localy. Otherwise, it is difficult to keep an overview. But in the future, I will leave it to the reviewer then.

Reply:
Please leave them to the reviewer :)